src/util-debug.c

```diff
@@ -445,6 +448,29 @@ static SCError SCLogMessageGetBuffer(
                 substr++;
                 break;
 
+            case SC_LOG_FMT_SUBSYSTEM:
+                temp_fmt[0] = '\0';
+                const char *fmt_string = "%s%s%s%s";
+                if (_sc_subsystem || module) {
+                    if (_sc_subsystem && module) {
+                        fmt_string = "%s%s[%s:%s]%s";
```

Comment:
I think the [ ] should not be part of this here. It can be added in the default and/or user supplied format string instead.

Reply:
Will move to the default log formats.

Reply:
Brackets moved to default log format string.